include/dpp/dpp.h

```diff
@@ -81,11 +81,11 @@ class collected_reaction : public managed {
 	/// Reacting user
 	user react_user;
 	/// Reacting guild
-	guild* react_guild;
+	guild* react_guild{};
```

Comment:
as these are pointers they should probably initialise to nullptr explicitly. {} doesn't do this right?

Reply:
brace initialization initializes pointers to `nullptr`.